nep-28.mediawiki

```diff
@@ -1,9 +1,11 @@
-NEP: 28
-Title: Standardization of the Smallest Unit of GAS
-Author: Jimmy Liao <jimmy@r3e.network>
-Type: Standard
-Status: Draft
-Created: 2024-05-20
+<pre>
+    NEP: 28
+    Title: Standardization of the Smallest Unit of GAS
+    Author: Jimmy Liao <jimmy@r3e.network>
```

Comment:
If it is HTML, this should be escaped?

Reply:
All the others are formatted like this and it seems to be OK: https://github.com/neo-project/proposals/blob/master/nep-16.mediawiki